### DISCUSSION TITLE
Versioning of anvil-labs

### DISCUSSION THREAD

**Discussion by steininge** [anvilistas/anvil-labs#148] [Ideas]
Would it make sense to publish anvil-labs under version numbers, just like anvil-extras? 

Then a production app could depend on a certain version of the anvil-labs and not break when there are changes.

**Comment by meatballs** [comment]
Not really. Labs is entirely experimental in nature and liable to break. We make that very clear.

If you want to depend on it in a production app, we suggest you clone your own copy and manage that dependency yourself.

**Reply by steininge** [reply]
OK, that makes sense 👍 I made my own clone.
